modules/c/nrt/source/IOInterface.c

```diff
@@ -200,7 +200,7 @@ NRTPRIV(NRT_BOOL) BufferAdapter_read(NRT_DATA * data, void *buf, size_t size,
 
     if (size > 0)
     {
-        memcpy(buf, (char *) (control->buf + control->mark), size);
+        memcpy(buf, (char *)(control->buf + control->mark), size);
```

Comment:
I know this pre-dates your changes but ditch the `char*` cast and parens here - `memcpy()` takes in a `void*` so we don't need to cast